File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Affix.java
package fr.jbme.raiderioapp.data.model.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Affix {

    @SerializedName("id")
    @Expose
    public Integer id;
    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("description")
    @Expose
    public String description;
    @SerializedName("wowhead_url")
    @Expose
    public String wowheadUrl;

}

File: app/src/main/java/fr/jbme/raiderioapp/model/RIOCharacter/Affix.kt
package fr.jbme.raiderioapp.model.RIOCharacter

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Affix(
    @SerializedName("id")
    @Expose
    var id: Int? = null,

    @SerializedName("name")
    @Expose
    var name: String? = null,

    @SerializedName("description")
    @Expose
    var description: String? = null,

    @SerializedName("wowhead_url")
    @Expose
    var wowheadUrl: String? = null
)